src/checkqueue.h

```diff
@@ -143,6 +143,9 @@ class CCheckQueue
         }
     }
 
+    CCheckQueue(const CCheckQueue&) = delete;
+    CCheckQueue& operator=(const CCheckQueue&) = delete;
```

Comment:
Should apply rule-of-5 and default the move operations, no?

Reply:
Thanks! [Done](https://github.com/bitcoin/bitcoin/pull/26762#issuecomment-1369886008).